golem.yaml

```
@@ -72,6 +74,105 @@ subpackages:
             set -e
             test "$(readlink /golem-component-service)" = "/usr/bin/golem-component-service"
 
+
+  - name: ${{package.name}}-component-compilation-service
+    dependencies:
+      runtime:
+        # https://github.com/golemcloud/golem/blob/main/golem-component-compilation-service/docker/Dockerfile#L19
+        - ca-certificates-bundle
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.contextdir}}/
+          mkdir -p ${{targets.contextdir}}/config
+          mkdir -p ${{targets.contextdir}}/db
+          mv ./target/release/golem-component-compilation-service ${{targets.contextdir}}/usr/bin/golem-component-compilation-service
```

```suggestion
          install -Dm0755 ./target/release/golem-component-compilation-service ${{targets.contextdir}}/usr/bin/golem-component-compilation-service
```
See: https://github.com/wolfi-dev/os/pull/55410/files#r2129056133